### DISCUSSION TITLE
Nicknames + Groups to Nametag

### DISCUSSION THREAD

**Discussion by isitreallyalive** [EssentialsX/Essentials#4258] [Ideas and Feature Suggestions] [answered]
I think it would be a nice feature if nicknames + prefixes were displayed in player nametags to avoid confusion as to who is who, as there is a setting to show nicknames in the tab list but that may not necessarily reflect what is displayed in the name tag. It would also allow people to quickly identify the roles of certain members.

**Comment by pop4959** [comment, marked as answer]
If you're talking about EssentialsChat I believe there are already placeholders that allow you to do this (for either the player name or display name -- display name being nicked).
